README.md

```diff
@@ -14,6 +14,7 @@ Movie search site that uses [OMDb API](http://omdbapi.com/) to get movies from I
 
 ```
 npm install
+go to omdb, get an apikey and paste it to the apiKey.js in the project src folder.
```

Comment:
Can you also add example file here because I have no idea what kind of file `apiKey.js` should be 🤔

Reply:
Example file? I don't understand how these instructions are unclear?

Reply:
Point is that you tell me `get an apikey and paste it to the apiKey.js` but it does not work if I add apiKey.js which has content like `lolapikeyhere`